Wireframe/index.html

```diff
@@ -4,9 +4,57 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
+    <article>
+    <section>
```

Comment:
Why did you wrap the header and main in this section? Can you talk me through it?